packages/braid-design-system/src/lib/components/TooltipRenderer/TooltipRenderer.tsx

```diff
@@ -72,10 +72,10 @@ export const TooltipContent = ({
   >
     <Box
       boxShadow="large"
-      background="customDark"
+      background="neutral"
```

Comment:
Does this affect dark mode in any way? Text legibility/contrast?

Reply:
Nope, exact some colour under the covers and "neutral" also changes the foreground context to use inverted text.